ogame/__init__.py

```diff
@@ -298,6 +298,28 @@ def celestial_coordinates(self, id):
                 coordinates.append(const.destination.moon)
                 return coordinates
 
+    def techtree(self, id):
```

Comment:
as mention in Discord we can save it offline and save the result in constnts.py

Reply:
yes, but in another class, i don't want to break your code ^^